Templates/Multiplayer/template.json

```diff
@@ -1071,10 +899,6 @@
             "file": "Prefabs/Player.prefab",
             "isTemplated": true
         },
-        {
```

Comment:
This file is still listed in [repo.json](https://github.com/GaianHelmers/o3de-extras/blob/303c9b5a0a8a2654d46254a577cf26a3a320a455/repo.json#L1488). I am aware this file is outdated, just saying...

Reply:
Good catch! Since the repo’s root repo.json won’t match, this will break the ability to download the MPTemplate directly inside of project manager (O3DE.exe). Because MPTemplate is already broken, I propose  merging this fix as a stepping stone for users who want to pull MPT via git, and then make another git hub issue for fixing the repo.json and release zip copy of mp template.

Reply:
Ah darn. I manually edited the template file as I'm not actually familiar with how to re-templatize the project.

I agree with the stepping stone sentiment, as it's looking like there's a Gem configuration issue with the Template that needs fixing too. Discord discussion: https://discord.com/channels/805939474655346758/816043601754325023/1248334270313791600